utils/importer.go

```diff
@@ -98,6 +98,10 @@ func (importer *universalImporter) Import(importedFrom, importedPath string) (js
 	var tried []string
 	for _, u := range candidateURLs {
 		foundAt := u.String()
+		// Avoid collision bug when importing same chart in the same jsonnet file using `import binary://` and `importbin`
+		if binary {
+			foundAt = "binary-" + u.String()
```

Comment:
what happens if the imported file name starts with `binary-` ?

Reply:
it breaks, let me see to do it more cleanly by using the original binary:// prefix

Reply:
i changed it to a suffix like `##binaryImport` to the filename. i think the risk of someone files being called `CHARTNAME.tgz##binaryImport` are close to 0 :)